src/main/webapp/app/components/exercise/difficulty-picker/difficulty-picker.component.html

```diff
@@ -2,23 +2,23 @@
     <div class="btn"
          [ngClass]="{'btn-primary': !exercise.difficulty,
                             'btn-default': exercise.difficulty}"
-         (click)="exercise.difficulty = undefined">
+         (click)="setDifficulty(undefined)">
```

Comment:
why do we need these changes? Aren't they unrelated to the Markdown Editor? At least comment on them in the ts file.